avaframe/com1DFA/com1DFA.py

```diff
@@ -1888,20 +1891,25 @@ def trackParticles(cfgTrackPart, dem, particlesList):
     return particlesList, trackedPartProp, track
 
 
-def readFields(inDir, resType, simName='', flagAvaDir=True, comModule='com1DFA'):
-    """ Read ascii files within a directory and return List of dicionaries
+def readFields(inDir, resType, simName='', flagAvaDir=True, comModule='com1DFA', timeStep=''):
+    """ Read ascii files within a directory and return List of dictionaries
 
         Parameters
         -----------
         inDir: str
             path to input directory
+        resType: list
+            list of desired result types
         simName : str
             simulation name
         flagAvaDir: bool
             if True inDir corresponds to an avalanche directory and pickles are
             read from avaDir/Outputs/com1DFA/particles
         comModule: str
             module that computed the particles
+        timeStep: float
```

Comment:
How does this have to be specified? Does it have to be precise or is a range considered? -> Provide info here

Reply:
I added an atol of 1.e-6 as the file name where the time info is read if saved from com1DFA is set to %.2f so could even be less